content/practices/ipm/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Iteration Planning Meetings (IPM)"
+linkTitle: "Iteration Planning Meetings (IPM)"
+description: "A regular meeting for the core team to understand and align on the work to be done."
+# Note: remove any tags that are not relevant.
+tags: ["Delivery"]
+length: "1 Hour"
+participants: "Core delivery team"
+# custom "cover" image example: "boris/boris.png"
+image: "default-cover.png" 
+lastmod: "2021-06-17"
+date: "2021-06-17"
+why: 
+- Regular planning meetings help ensure the product backlog is well-understood by all team members and always reflects the current priorities. By discussing and sizing product backlog items, the team may align on the delivery impact of the work to be done.
+when:
+- Iteration Planning Meetings should generally follow the cadence of product iterations (e.g. weekly) or should be held as often as needed to maintain a well-sized and well-understood product backlog.
+what:
+- A product backlog
+- A prepared set of prioritized user stories to be reviewed and potentially promoted to the product backlog
+
+# If this practice or workshop has a Miro template: remote: true
+remote: false
+miro_template_url: "URL for related Miro template" 
+
+---
+## How to Use this Method
+### Sprint Planning vs. Iteration Planning
```

Comment:
Looks like this header line should be moved down to Line 85 (under "Related Practices" and above the handy table comparing Sprints vs. Iterations).

Reply:
Good catch!

Reply:
Fixed